kolibri/core/tasks/storage.py

```diff
@@ -329,10 +329,10 @@ def restart_job(self, job_id):
         Raises `JobNotRestartable` exception if the job with id = job_id state is
         not in CANCELED or FAILED.
         """
-        with self.session_scope() as session:
-            job_to_restart, orm_job = self._get_job_and_orm_job(job_id, session)
-            queue = orm_job.queue
-            priority = orm_job.priority
+        orm_job = self.get_orm_job(job_id)
+        job_to_restart = self._orm_to_job(orm_job)
+        queue = orm_job.queue
+        priority = orm_job.priority
 
```

Comment:
Since we are using connection-based database reads. Now, we don't need to assign `queue` and `priority` as variables. We can directly use `orm_job.queue` and `orm_job.priority` in this function's code.

Reply:
yup! should be.